ratel-visitor/src/expression.rs

```diff
@@ -52,6 +52,10 @@ impl<'ast> Visitable<'ast> for ExpressionNode<'ast> {
                 visitor.push_parent(ParentNode::from(self), ctx);
                 computed.traverse(visitor, ctx);
             },
+            MetaProperty(ref property) => {
+                property.traverse(visitor, ctx);
+                visitor.on_meta_property(property, self, ctx);
```

Comment:
After the parent tracking PR, the order here should be:

1) `on_meta_property`
2) `push_parent` (new)
3) `traverse`